template/{{.input_root_dir}}/docs/mlops-setup.md.tmpl

```diff
@@ -114,6 +122,20 @@ to add the secrets to GitHub:
 - `PROD_WORKSPACE_TOKEN` : service principal token for prod workspace
 - `WORKFLOW_TOKEN` : [Github token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic) with workflow permissions. This secret is needed for the Deploy CI/CD Workflow.
 
+Next, be sure to update the [Workflow Permissions](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#modifying-the-permissions-for-the-github_token) section under Repo Settings > Actions > General:
```

Comment:
Same as above, combine redundant lines please!

Reply:
@arpitjasa-db simple question for .tmlp is or written like `{{ if and (eq .input_cicd_platform `github_actions`) (eq .input_cloud `aws`) or (eq .input_cloud `gcp`)}}`